views/basket/index.php

```diff
@@ -100,6 +100,8 @@
 
                 $this->load->view($skin->path . 'views/basket/table', $tableData);
 
+                $this->load->view($skin->path . 'views/basket/mobile');
```

Comment:
Can you check if we need to pass $tableData in to this view? If not, can you check whether or not it is required for the table view too? Seems like one must be wrong!